src/MessageHandler.py

```diff
@@ -0,0 +1,36 @@
+import urllib2
+from src.CastsCommandHandler import CastsCommandHandler
+from src.MessageRouter import MessageRouter
+from src.StreamsCommandHandler import StreamsCommandHandler
+
+
+class MessageHandler:
+    """
+    This class is responsible for constructing the object graph needed
+    to handle messages and to delegate to it.
+    """
```

Comment:
Is that still the case for https://github.com/FAForever/fafbot/commit/d1719067bead1dde8fbdb8022e109785f95727ae? I do not really understand what your suggestion would look like. What would be better, design wise, of using a function? Keep in mind that this will grow as more commands are added. In the linked commit the class has 2 private methods. If the function approach was used, would one not be forced to stuff all the things in a single function, or to have multiple functions in the file scope?